packages/jest-jasmine2/src/reporter.js

```diff
@@ -4,39 +4,71 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
  */
 'use strict';
 
+import type {Config} from 'types/Config';
```

Comment:
Wait, I thought you were going to create a type in this file `type JasmineReporterConfig = {
noHighlight, noStackTrace};` so we don't pass the entire config along here. Sorry if what I said wasn't clear.